corehq/apps/domain/views/base.py

```diff
@@ -86,7 +87,7 @@ def dispatch(self, *args, **kwargs):
         return super(LoginAndDomainMixin, self).dispatch(*args, **kwargs)
 
 
-class BaseDomainView(LoginAndDomainMixin, BaseSectionPageView, DomainViewMixin):
+class BaseDomainView(LoginAndDomainMixin, DowngradeModalMixin, BaseSectionPageView, DomainViewMixin):
```

Comment:
So this is going to show up on the dashboard and then on many but not all other pages - like I think it'll show on SMS pages but it won't show in app manager. Is that deliberate? I'm wondering if it'd make sense to show it on a smaller but very deliberate set of pages.

Reply:
We're trying to include this on as many user-facing pages as is realistically possible.  Will it not show in app manager because the app manager doesn't inherit from ```BaseDomainView```?

Reply:
Yes, exactly, it won't show on any function-based view. They're a minority; I think they also include exports and case importer. If you want this on every page, the way to do it is to add the context to `context_processors.py` or to the request object (though I think it's better to bloat the global context than the request object). That's what we do for other global content like the EULA and trial modals.